File: src/main/java/com/tina/Game1.java
package com.tina;

import java.util.Random;
import java.util.Scanner;

public class Game {
    public static void main(String[] args) {
        /*Random random = new Random();
        int secret = random.nextInt(10)+1;*/
        //上面兩行程式碼＝下面一行
        int secret = new Random().nextInt(10)+1;
        System.out.println(secret);
        Scanner scanner = new Scanner(System.in);
        int number = 0;
        while (number != secret){
            System.out.println("請輸入數字：");
            number = scanner.nextInt();
            if(number>secret) {
                System.out.println("小一點");
            } else if (number<secret) {
                System.out.println("大一點");
            } else{
                System.out.println("bingo");
                break;
            }



        }

    }
}

File: src/main/kotlin/com/tina/Game1.kt
package com.tina

import java.util.*


fun main(args: Array<String>) {
    val secret = Random().nextInt(10)+1
    println(secret)
    val scanner = Scanner(System.`in`)
    var number = 0
    while (number != secret) {
        print("請輸入數字:")
        //number = scanner.nextInt()
        //可改成
        number = readLine()!!.toInt()
        if (number > secret) {
            println("lower")
        } else if (number < secret)
            println("higher")
        else {
            println("bingo!!!")
        }
    }
}
